### PR TITLE
set up circleci

### PR DESCRIPTION
**Pull request by leejh903**
## Proposed changes

- This change can run integration tests and unit tests in modules when new commit has created.
- It releases new version to bintray when a tag has made.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-java/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

none

## Further comments

none

### CODE REVIEW

File: .circleci/images/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        --metrics \
+        --prometheus \
+        --syncmode "full" \
+        --rewardbase 0x73718c4980728857f3aa5148e9d1b471efa3a7dd
+
+    networks:
+      app_net:
+        ipv4_address: 172.16.239.10
+    restart: "no"
+  prometheus:
```
Comment (kjhman21) [inline .circleci/images/docker-compose.yml]: I don't think we really need to run prometheus for CI. Please remove this section and the file `prometheus.yml`.

File: .circleci/images/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    ports:
+      - 9090:9090
+    volumes:
+      - "./prometheus.yml:/etc/prometheus/prometheus.yml"
+    networks:
+      app_net:
+        ipv4_address: 172.16.239.9
+    restart: "no"
+
+  grafana:
```
Comment (kjhman21) [inline .circleci/images/docker-compose.yml]: Same here. Please remove this section.

File: .circleci/images/docker-compose.yml (hunk 1 above)
Comment (leejh903) [reply .circleci/images/docker-compose.yml]: I think so!! Thanks for letting me know.

File: .circleci/images/docker-compose.yml (hunk 2 above)
Comment (leejh903) [reply .circleci/images/docker-compose.yml]: Thanks. I will remove that section.

**Comment by kjhman21** [comment]
We do not need images/Klaytn.json  and images/Klaytn_txpool.json. Please remove them, too.

**Pull request merged**